dockerfiles/Dockerfile-delegation-backend

```diff
@@ -5,13 +5,15 @@ WORKDIR $GOPATH/src/delegation_backend
 
 # Copy everything from the current directory to the PWD (Present Working Directory) inside the container
 COPY src src
+COPY database ../../../database
```

Comment:
Why keep the path relative if you're just copying database to /database? I.e.
`COPY database /database `

Reply:
Indeed.